spec/features/consumer/shopping/unit_price_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'spec_helper'
+
+feature "As a consumer, I want to check unit price information for a product ;", js: true do
+  include AuthenticationHelper
+  include WebHelper
+  include ShopWorkflow
+  include UIComponentHelper
+
+  # Create context
```

Comment:
IMO this comment doesn't bring any value. That's what `let`s do after all.

Reply:
👌